File: 2020_1/ead-apc/ex4.py
#  entre 0 e 5 = reprovado
#  >= 5 < 7 = mm
#  >= 7 < 9 = ms
#    >= 9 <= 10 = ss

nota = 9.5

if nota < 5:
    print("Reprovado")
else:
    if (nota >=5 and nota <7):
        print("MM")
    else:
        if (nota >=9):
            print("SS")
        else:
            if (nota>=7 and nota <9):
                print("MS")


if nota < 5:
    print("Reprovado")
elif (nota >=5 and nota <7):
    print("MM")
elif (nota >=9):
    print("SS")
elif (nota>=7 and nota <9):
    print("MS")



    

    

File: 2021_2/apc2/ex4.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    float nota;

    printf("\nDigite sua nota: ");
    scanf("%f",&nota);

    if (nota<0 || nota > 10)
    {
        printf("\n Nota invalida");
    }

    if (nota >=0 && nota <5)
    {
        printf("\n Reprovado");
    }

    if (nota >=5 && nota<=10)
    {
        printf("\n Aprovado");
    }
}